packages/neo-one-website/docs/2-advanced-guides/10-configuration-options

```diff
@@ -0,0 +1,41 @@
+---
+slug: config options
+title: Configuration Options
+---
+NEO•ONE compiler configuration options
+
+Configure NEO•ONE for your project.
+
+## Config File
+
+An [initialized environment](https://neo-one.io/tutorial#Setup-for-the-Tutorial) will have a ` .onerc ` file within the project root.
+
+### `codegen`
+
+Keys with `codegen` allow you to configure which `language`, front-end `framework` and whether to generate code for the `browser`.
+* `language` may be  `typescript` or `javascript`
+* `framework` currently supports choosing one of `none`, `react`, `vue` and `angular`
+* `browser` is a boolean; either `true` or `false`
+
+### `paths`
```

Comment:
This should be changed to match the style of `codegen` . Something like: "
There are two options within `path` which defined where NEO•ONE will look for smart contracts to compile and where it will output generated code.  
*`contracts`: `string`.  Path at which NEO•ONE will search for smart contracts to compile. Defaults to `one/contracts`.
*`generated`: `string`. Path at which NEO•ONE will output generated code.  Defaults to `one/generated`.
"

Reply:
very helpful, thank you!